pkg/appregistry/adapter.go

```diff
@@ -29,9 +31,13 @@ type apprApiAdapterImpl struct {
 	client *appr.Appregistry
 }
 
-func (a *apprApiAdapterImpl) ListPackages() (appr_models.Packages, error) {
+func (a *apprApiAdapterImpl) ListPackages(namespace string) (appr_models.Packages, error) {
 	params := appr_package.NewListPackagesParams()
 
+	if namespace != "" {
```

Comment:
Not a huge fan of these sort of interfaces. I would prefer having `ListPackages()` in addition to this API.

Reply:
I think logging it here and adding some message to end-user is really useful because.